app/src/main/java/de/westnordost/streetcomplete/osm/Maxspeed.kt

```diff
@@ -10,6 +10,18 @@ val MAXSPEED_TYPE_KEYS = setOf(
     "zone:traffic"
 )
 
+val anyMaxSpeedTagKey = "~${(MAXSPEED_TYPE_KEYS + "maxspeed").joinToString("|")}"
+
+val explicitlyInSlowZone = """$anyMaxSpeedTagKey ~ ".*:(zone)?:?([1-9]|[1-2][0-9]|30)""""
+
+// This assumes that elements only tagged with 'maxspeed' and no other maxspeed tags are in a slow zone.
```

Comment:
Hmm... that's rather bold, and **eurocentric**. And even within (Germany), there are plenty of tempo-30 roads that are not 30 zones.

Slow zones as a concept do not exist in that many countries, and hence, the associated legislation does also not exist. I.e. in this case, that the local traffic authority may only designate roads as within 30 zones if there is no compulsory cycleways.

Incidentally, the source:maxspeed tagging in Germany (and Europe) is most complete when compared to the rest of the world, so I think even the assumption that if maxspeed=30 it must be a 30-zone will often be wrong even in Germany / Europe.

Reply:
So the naming and comment is incorrect. Or do you propose to change the filter and the tests?
I'm not sure how to proceed.

Reply:
Well, I wouldn't like this behavior: That simply a `maxspeed=30` is assumed to be a 30-zone and this means that surely there'll never be bike paths then.

Reply:
My comment is wrong, so I removed it.